src/storage/test/StorageIndexWriteBenchmark.cpp

```diff
@@ -119,9 +219,9 @@ void insertVertices(bool withoutIndex) {
             }
```

Comment:
line 103 ~ line 119 
This part of the code can be proposed separately
Avoid code duplication and Can be reused